### PR TITLE
fix: clean code & fix bugs

### PR DESCRIPTION
**Pull request by danielsp45**

### CODE REVIEW

File: src/components/splash.astro [hunk 1]
```diff
@@ -2,11 +2,13 @@
 import { Image } from "astro:assets";
 import { Icon } from "astro-icon/components";
 import gubImage from "~/assets/Bicho.svg";
+import background from "/public/background.jpg"
 ---
 
 <section class="relative h-full bg-black overflow-y-hidden select-none">
   <div id="splash-bg-fallback" class="absolute inset-0 hidden opacity-40">
-    <img src="/background.jpg" class="h-full w-full object-cover" />
+    <!-- <img src="/background.jpg" class="h-full w-full object-cover" /> -->
```
Comment (MarioRodrigues10) [inline src/components/splash.astro:10]:
```suggestion

```

**Review by MarioRodrigues10 — CHANGES_REQUESTED** [review]

**Comment by MarioRodrigues10** [comment]
I think it would be more interesting to change the text to English. 

What's your thoughts on it @danielsp45 ?

**Review by joaodiaslobo — APPROVED** [review]
🪲💚

**Review by MarioRodrigues10 — APPROVED** [review]
💚

**Pull request merged**